statvar_imports/worldbank_ids/configs/currency_pvmap.csv

```diff
@@ -0,0 +1,47 @@
+key,prop,value,p1,v1,,,,,,
+year,observationDate,{Data},measuredProperty,amount,unit,Percent,measurementDenominator,-currency,populationType,Debt
+value,value,{Number},#Eval,"measurementMethod=""WorldBankProjection"" if Year>""2022-01-01"" else """"",,,,,,

```

Comment:
Pls add a note to readme if such thresholds have to be modified for each run.

Could also convert it into a variable, or better still, set it using an eval formula:
Eval,LastYear=datetime.now().year - 2

then use the referenc ein the formula
"measurementMethod=""WorldBankProjection"" if Year>""{LastYear}"" else """""